packages/pymedphys_dicom/src/pymedphys_dicom/dicom/anonymise.py

```diff
@@ -534,7 +534,7 @@ def unknown_tags_in_dicom_dataset(ds):
         try:
             get_baseline_dict_entry(tag)
             are_non_private_tags_in_dict_baseline.append(True)
-        except NotInBaseLineDictError:
+        except NotInBaselineDictError:
```

Comment:
Removing `Dict`?

Reply:
Not sure what happened there sorry mate. Pushing up changes now.

Reply:
All good :)